src/app/home-page/recent-events/recent-events.component.html

```diff
@@ -18,43 +18,37 @@
         <ng-template #defaultAvatar>
           <img [src]="gravatarService.gravatarUrlForMysteryPerson()" alt="User avatar" />
         </ng-template>
+        <ng-template #withUsername>
+          <strong>
+            <a class="no-underline" [routerLink]="'/users/' + event.initiatorUser?.username">
+              {{ event.initiatorUser?.username }}
+            </a>
+          </strong>
+        </ng-template>
+        <ng-template #withoutUsername>
+          <a class="no-underline"> Unknown user </a>
```

Comment:
I think you can remove the `<a>` element because there's no link

Reply:
You could also add `<strong>` so "Unknown user" is bolded to match the bolded usernames